integrationtest/showcase/library/pom.xml

```diff
@@ -37,6 +37,20 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <!--
+                NOTE: spring brings its versions of the elasticsearch client.
+                We want to test on the version we are using hence we override it here:
+             -->
```

Comment:
Shouldn't we just override the property `elasticsearch.version`, like we do with `hibernate.version` above, and like we [recommend in the documentation](https://docs.jboss.org/hibernate/stable/search/reference/en-US/html_single/#compatibility-framework-spring-boot-dependency-versions)?

Reply:
Good point 😃 
It looks like they've changed the property name in the new version of their bom, so I've updated the doc and added a tip.

Reply:
Looks like you didn't remove dependency management?

Reply:
hmm, so I've just tested it once again, and seems like this approach doesn't work for us:
```
[INFO] --- maven-dependency-plugin:3.6.0:tree (default-cli) @ hibernate-search-integrationtest-showcase-library ---
[INFO] org.hibernate.search:hibernate-search-integrationtest-showcase-library:jar:7.0.0-SNAPSHOT
[INFO] +- org.hibernate.search:hibernate-search-mapper-orm:jar:7.0.0-SNAPSHOT:compile
[INFO] |  +- org.hibernate.search:hibernate-search-engine:jar:7.0.0-SNAPSHOT:compile
[INFO] |  +- org.hibernate.search:hibernate-search-mapper-pojo-base:jar:7.0.0-SNAPSHOT:compile
[INFO] |  +- org.hibernate.search:hibernate-search-util-common:jar:7.0.0-SNAPSHOT:compile
[INFO] |  +- org.hibernate.orm:hibernate-core:jar:6.2.5.Final:compile
[INFO] |  \- org.hibernate.common:hibernate-commons-annotations:jar:6.0.6.Final:compile
[INFO] +- org.hibernate.search:hibernate-search-backend-lucene:jar:7.0.0-SNAPSHOT:compile
[INFO] +- org.hibernate.search:hibernate-search-backend-elasticsearch:jar:7.0.0-SNAPSHOT:compile
[INFO] |  +- org.elasticsearch.client:elasticsearch-rest-client:jar:8.7.1:compile
[INFO] |  \- org.elasticsearch.client:elasticsearch-rest-client-sniffer:jar:8.7.1:compile
[INFO] +- org.hibernate.search:hibernate-search-backend-elasticsearch-aws:jar:7.0.0-SNAPSHOT:test
[INFO] \- org.hibernate.search:hibernate-search-util-internal-integrationtest-mapper-orm:jar:7.0.0-SNAPSHOT:test
[INFO]    +- org.hibernate.search:hibernate-search-util-internal-test-orm:jar:7.0.0-SNAPSHOT:test
[INFO]    |  \- org.hibernate.search:hibernate-search-util-internal-test-common:jar:7.0.0-SNAPSHOT:test
[INFO]    +- org.hibernate.search:hibernate-search-util-internal-integrationtest-common:jar:7.0.0-SNAPSHOT:test
[INFO]    \- org.hibernate.orm:hibernate-testing:jar:6.2.5.Final:test
[INFO]       \- org.hibernate.orm:hibernate-community-dialects:jar:6.2.5.Final:test
```
So I've tried to do it on a spring boot project and it worked  fine there ... Looking into it -- I suppose the reason is because we are importing the pom (so I guess the versions/properties at that point are resolved?) while in case of a boot project these dependencies/properties/versions get to the user pom through parents ...

So I'm not sure... we could leave it as is and use whatever the versions are in boot, or force it with an override like the one I haven't removed 🙈 😃

Reply:
Ah, it's possible that overriding properties only works when using Spring Boot's parent POM, and not when using the BOM.

If true, we'd need to add dependency management for ORM as well...

Or actually, we could just import the Hibernate Search public BOM just after Spring Boot's?

Could you look into this and update the documentation accordingly?